src/controllers/submit.controller.ts

```diff
@@ -44,8 +44,10 @@ function renderSubmitPage(req: SubmitPageRequest, res: Response) {
     });
 }
 
-function validateForm(file?: Express.Multer.File) {
-    const validationResult = new ValidationResult();
+function validateForm(file?: Express.Multer.File, validationResult?: ValidationResult) {
+    if (validationResult === undefined) {
+        validationResult = new ValidationResult();
+    }
```

Comment:
Could use a default parameter for this:
```suggestion
function validateForm(file?: Express.Multer.File, ValidationResult = new ValidationResult()) {
```